might be good to add "_readthedocs" to the .gitignore file

Also, linter wants

```
./.readthedocs.yaml
  4:1       warning  missing document start "---"  (document-start)
```

because it's a yaml file, it's checked by yamllint. If that might cause trouble there is an ignore list somewhere.

--- a/.gitignore
+++ b/.gitignore
@@ -166,5 +166,6 @@ use_dev_supervisor.txt
 awx/ui_next/src
 awx/ui_next/build
 
-# Docs build
+# Docs build stuff
 docs/docsite/build/
+_readthedocs/

--- a/.yamllint
+++ b/.yamllint
@@ -10,6 +10,7 @@ ignore: |
   tools/docker-compose/_sources
   # django template files
   awx/api/templates/instance_install_bundle/**
+  .readthedocs.yaml
 
 extends: default
 

